app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user, foreign_key: :author_id
```

Comment:
- [ ] According to the ERD, the foreign key in this post table should be AuthorID, therefore the association should be:
`belongs_to :author, class_name: 'User'`

Please consider refactoring this appropriately.